STYLEGUIDE.md

```diff
@@ -322,6 +325,99 @@ function changeValues(object) {
 
 ---
 
+## <a id="Forwarding-Refs">Forwarding refs</a>
+
+Ref forwarding allows us to provide direct access to the underlying parent element. For more information on ref forwarding, check out the React [docs](https://reactjs.org/docs/forwarding-refs.html).
+
+#### Prefer
+
+```typescript
+const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
+  ({ children }: ButtonProps, forwardedRef) => (
+    <button ref={forwardedRef}>{children}</button>
+  ),
+);
+
+// This ref will give you direct access to the DOM button
+const ref = useRef<null | HTMLButtonElement>(null);
+<Button ref={ref}>Click me!</Button>;
+```
+
+#### Avoid
+
+```typescript
+const Button = props => <button>{props.children}</button>;
+
+// Does not accept a ref
+<Button>Click me!</Button>;
+```
+
+---
+
+## <a id="Consuming-Darkmode-From-Leafygreenprovider">Consuming darkMode from `LeafyGreenProvider`</a>
+
+The `useDarkMode()` hook allows components to read the value of the `darkMode` prop from the LeafyGreen provider and overwrite the value locally if necessary.
```

Comment:
ditto here

Reply:
also nit I think we should caps `Provider`

Reply:
can you add a sentence abut why we re-instantiate a provider in composite components

Reply:
another small nit - we should use the \` quotes around `darkMode`

Reply:
@brookescarlett is the section below this one, `Passing darkMode to children`, a good summary of why we re-instantiate a provider?